### PR TITLE
Allow to change the default zone

### PR DESCRIPTION
**Pull request by imobachgs**
This PR allows to change the Firewalld's default zone.

change-default-zone

### CODE REVIEW

File: src/lib/y2firewall/widgets/zones_table.rb [hunk 1]
```diff
@@ -32,22 +32,74 @@ class ZonesTable < ::CWM::Table
...
+        default_zone_button.zone = selected_zone
       end
 
       # @see CWM::Table#header
       def header
         [
           _("Name"),
-          _("Interfaces")
+          _("Interfaces"),
+          _("Default")
```
Comment (teclator) [inline src/lib/y2firewall/widgets/zones_table.rb]: I'm not sure about having an extra column for it, but probably that is something for Ken

Comment (imobachgs) [reply src/lib/y2firewall/widgets/zones_table.rb]: We have something similar in the repository list. But we might append a "(default)" string if preferred.

**Comment by teclator** [comment]
LGTM once travis is fine with the rdoc check :wink:

**Comment by imobachgs** [comment]
> LGTM once travis is fine with the rdoc check wink

Thanks :)

**Pull request merged**